modules/400-descheduler/openapi/config-values.yaml

```diff
@@ -51,13 +51,11 @@ properties:
       type: string
     description: |
       The same as the `spec.nodeSelector` pod parameter in Kubernetes.
```

Comment:
Some configuration options are deprecated, I assume.

Reply:
I believe, I have to preserve those for one deckhouse release. To allow the migration hook to complete.